src/directives/formly-form.js

```diff
@@ -16,7 +16,7 @@ module.exports = ngModule => {
       restrict: 'E',
       template: function(el, attrs) {
         /* jshint -W033 */ // this because jshint is broken I guess...
-        var rootEl = attrs.rootEl || 'ng-form';
+        var rootEl = attrs.rootEl || 'form';
```

Comment:
Definitely don't want to have it change this kind of behavior in different browsers. That could lead to some really odd bugs. Why couldn't we simply add `'ng-form'` to the list of custom elements in the `formlyCustomTags` file you've made?